## Summary 

| File | Release | Date | Source |  
| :--- | :--- | :--- | :--- |
| CCLE_RNAseq_genes_counts_20180929.gct.gz | CCLE 2019 | 09/18 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RNAseq_genes_counts_20180929.gct.gz) |
| CCLE_metabolomics_20190502.csv | CCLE 2019 | 05/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_metabolomics_20190502.csv) | 
| CCLE_RPPA_20181003.csv | CCLE 2019 | 01/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RPPA_20181003.csv) |
| CCLE_RPPA_Ab_info_20181226.csv | CCLE 2019 | 01/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RPPA_Ab_info_20181226.csv) |
| protein_quant_current_normalized.csv.gz | Proteomics | 01/20 | [GYGI](https://gygi.med.harvard.edu/sites/gygi.med.harvard.edu/files/documents/protein_quant_current_normalized.csv.gz) |
| Table_S1_Sample_Information.xlsx | Proteomics | 01/20 | [GYGI](https://gygi.med.harvard.edu/sites/gygi.med.harvard.edu/files/documents/Table_S1_Sample_Information.xlsx) |
| CCLE_mutations.csv | DepMap Public 20Q2 | 05/20 | [DepMap](https://ndownloader.figshare.com/files/22629110) |
> See below for details

---
### CCLE_RNAseq_genes_counts_20180929.gct.gz
#### File Description
CCLE RNAseq gene expression data fpr 1019 cell lines (read counts)

#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
> Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---
### CCLE_metabolomics_20190502.csv
#### File Description
CCLE metabolomics dataset (clean, imputed). Dataset of 225 metabolite levels in 928 cell lines from more than 20 cancer types in CCLE, as profiled using liquid chromatography–mass spectrometry (LC-MS).
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
> Haoxin Li, Shaoyang Ning, Mahmoud Ghandi, Gregory V. Kryukov, Shuba Gopal, ... Levi A. Garraway & William R. Sellers. The landscape of cancer cell line metabolism. Nature Medicine 25, 850-860 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### CCLE_RPPA_20181003.csv
* 214 Antibodies
* 899 Cell Lines
* 28 Primary Diseases
* 28 Lineages
#### File Description
CCLE Reverse Phase Protein Array (RPPA) data
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### CCLE_RPPA_Ab_info_20181226.csv
#### File Description
RPPA antibody information
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### protein_quant_current_normalized.csv
* 12399 Proteins
* 375 Cell Lines
* 24 Primary Diseases
* 27 Lineages
#### File Description
This is the core table of normalized protein expression data for all cell lines
#### Proteomics
Quantitative profiling of thousands of proteins by mass spectrometry across 375 cell lines from the Gygi lab. The normalized protein quantitation data from https://gygi.med.harvard.edu/publications/ccle was imported into the portal. For more information about this dataset released in Nusinow et al., 2020, please see this [supplemental guide](https://www.biorxiv.org/content/10.1101/2020.02.03.932384v1).

#### Data Usage
Please cite the following when using these data
> David P. Nusinow, John Szpyt, Mahmoud Ghandi, Christopher M. Rose, E. Robert McDonald III, Marian Kalocsay, Judit Jané-Valbuena, Ellen Gelfand, Devin K. Schweppe, Mark Jedrychowski, Javad Golji, Dale A. Porter, Tomas Rejtar, Y. Karen Wang, Gregory V. Kryukov, Frank Stegmeier, Brian K. Erickson, Levi A. Garraway, William R. Sellers, Steven P. Gygi (2020). Quantitative Proteomics of the Cancer Cell Line Encyclopedia. Cell 180, 2. https://doi.org/10.1016/j.cell.2019.12.023

---

### CCLE_mutations.csv
* 18802 Genes
* 1741 Cell Lines
* 35 Primary Diseases
* 37 Lineages
#### File Description
MAF of gene mutations.

For all columns with AC, the allelic ratio is presented as [ALTERNATE:REFERENCE].

* CGA_WES_AC: the allelic ratio for this variant in Broad WES using a cell line adapted version of the CGA pipeline (https://docs.google.com/document/d/1VO2kX_fgfUd0x3mBS9NjLUWGZu794WbTepBel3cBg08/edit) that includes germline filtering.
* SangerWES_AC: in Sanger WES
* SangerRecalibWES_AC: in Sanger WES after realignment at Broad
* RNAseq_AC: in Broad RNAseq data from the CCLE2 project
* HC_AC: in Broad Hybrid capture data from the CCLE2 project
* RD_AC: in Broad Raindance data from the CCLE2 project
* WGS_AC: in Broad WGS data from the CCLE2 project

Additional columns:
 
* isTCGAhotspot: is this mutation commonly found in TCGA
* TCGAhsCnt: number of times this mutation is observed in TCGA
* isCOSMIChotspot: is this mutation commonly found in COSMIC
* COSMIChsCnt: number of samples in COSMIC with this mutation
* ExAC_AF: the allelic frequency in the Exome Aggregation Consortium (ExAC)

Descriptions of the remaining columns in the MAF can be found here: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/

#### Mutations
CCLE mutation calls are aggregated from several different sources and sequencing technologies.

Currently, WES-based calls are generated on a quarterly basis using a cell line implementation of the CGA WES Characterization Pipeline (more info here: https://docs.google.com/document/d/1VO2kX_fgfUd0x3mBS9NjLUWGZu794WbTepBel3cBg08/edit). The cell line implementation of this pipeline has one key difference: because CCLE lines do not have a matched control sample, we use the same normal sample as a matched normal input for all samples using the same baits (Agilent vs. ICE). In addition, mutation calls still use hg19 alignment.

Quarterly WES-based mutation calls are added to the existing mutation calls from previous releases. WGS, Sanger WES, RNAseq, HC, and RD based calls from the 2nd phase of the CCLE project are also included. More details on how these mutations were called and filtered can be found in the manuscript “Next generation characterization of the Cancer Cell Line Encyclopedia” in Nature.

#### DepMap Public 20Q2
This DepMap release contains data from CRISPR knockout screens from project Achilles, as well as genomic characterization data from the CCLE project.

#### Data Usage
Please cite the following when using these data
> Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

This data was generated in part from funding from
This project is partially funded by CTD2, the Achilles consortium, and The Carlos Slim Foundation in Mexico through the Slim Initiative for Genomic Medicine.

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).


