## Summary 

| File                                     | Release            | Date  | Source                                                                                                                  |
| :---                                     | :---               | :---  | :---                                                                                                                    |
| CCLE_RNAseq_genes_counts_20180929.gct.gz | CCLE 2019          | 09/18 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RNAseq_genes_counts_20180929.gct.gz)                                   |
| CCLE_metabolomics_20190502.csv           | CCLE 2019          | 05/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_metabolomics_20190502.csv)                                             |
| CCLE_RPPA_20181003.csv                   | CCLE 2019          | 01/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RPPA_20181003.csv)                                                     |
| CCLE_RPPA_Ab_info_20181226.csv           | CCLE 2019          | 01/19 | [CCLE](https://data.broadinstitute.org/ccle/CCLE_RPPA_Ab_info_20181226.csv)                                             |
| protein_quant_current_normalized.csv.gz  | Proteomics         | 01/20 | [GYGI](https://gygi.med.harvard.edu/sites/gygi.med.harvard.edu/files/documents/protein_quant_current_normalized.csv.gz) |
| Table_S1_Sample_Information.xlsx         | Proteomics         | 01/20 | [GYGI](https://gygi.med.harvard.edu/sites/gygi.med.harvard.edu/files/documents/Table_S1_Sample_Information.xlsx)        |
| CCLE_mutations.csv                       | DepMap Public 20Q2 | 05/20 | [DepMap](https://ndownloader.figshare.com/files/22629110)                                                               |
| CCLE_GCN.csv                             | DepMap Public 20Q2 | 05/20 | [DepMap](https://ndownloader.figshare.com/files/22629107)                                                               |
| Achilles_gene_effect.csv                 | DepMap Public 20Q2 | 05/20 | [DepMap](https://ndownloader.figshare.com/files/22629068)                                                               |
| sample_info.csv                          | DepMap Public 20Q2 | 05/20 | [DepMap](https://ndownloader.figshare.com/files/22629137)                                                               |
> See below for details

---
### CCLE_RNAseq_genes_counts_20180929.gct.gz
#### File Description
CCLE RNAseq gene expression data fpr 1019 cell lines (read counts)

#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
> Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---
### CCLE_metabolomics_20190502.csv
#### File Description
CCLE metabolomics dataset (clean, imputed). Dataset of 225 metabolite levels in 928 cell lines from more than 20 cancer types in CCLE, as profiled using liquid chromatography–mass spectrometry (LC-MS).
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
> Haoxin Li, Shaoyang Ning, Mahmoud Ghandi, Gregory V. Kryukov, Shuba Gopal, ... Levi A. Garraway & William R. Sellers. The landscape of cancer cell line metabolism. Nature Medicine 25, 850-860 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### CCLE_RPPA_20181003.csv
* 214 Antibodies
* 899 Cell Lines
* 28 Primary Diseases
* 28 Lineages
#### File Description
CCLE Reverse Phase Protein Array (RPPA) data
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### CCLE_RPPA_Ab_info_20181226.csv
#### File Description
RPPA antibody information
#### CCLE 2019
Expanded characterizations of CCLE cancer cell lines. This includes RNA sequencing, whole-exome sequencing, whole-genome sequencing, reverse-phase protein array, reduced representation bisulfite sequencing, microRNA expression profiling, global histone modification profiling, and metabolomics.

Processed data downloads are available here. Raw sequencing data is available through the Sequence Read Archive under accession number PRJNA523380.

#### Data Usage
Please cite the following when using these data
Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

---

### protein_quant_current_normalized.csv
* 12399 Proteins
* 375 Cell Lines
* 24 Primary Diseases
* 27 Lineages
#### File Description
This is the core table of normalized protein expression data for all cell lines
#### Proteomics
Quantitative profiling of thousands of proteins by mass spectrometry across 375 cell lines from the Gygi lab. The normalized protein quantitation data from https://gygi.med.harvard.edu/publications/ccle was imported into the portal. For more information about this dataset released in Nusinow et al., 2020, please see this [supplemental guide](https://www.biorxiv.org/content/10.1101/2020.02.03.932384v1).

#### Data Usage
Please cite the following when using these data
> David P. Nusinow, John Szpyt, Mahmoud Ghandi, Christopher M. Rose, E. Robert McDonald III, Marian Kalocsay, Judit Jané-Valbuena, Ellen Gelfand, Devin K. Schweppe, Mark Jedrychowski, Javad Golji, Dale A. Porter, Tomas Rejtar, Y. Karen Wang, Gregory V. Kryukov, Frank Stegmeier, Brian K. Erickson, Levi A. Garraway, William R. Sellers, Steven P. Gygi (2020). Quantitative Proteomics of the Cancer Cell Line Encyclopedia. Cell 180, 2. https://doi.org/10.1016/j.cell.2019.12.023

---

### CCLE_mutations.csv
* 18802 Genes
* 1741 Cell Lines
* 35 Primary Diseases
* 37 Lineages
#### File Description
MAF of gene mutations.

For all columns with AC, the allelic ratio is presented as [ALTERNATE:REFERENCE].

* CGA_WES_AC: the allelic ratio for this variant in Broad WES using a cell line adapted version of the CGA pipeline (https://docs.google.com/document/d/1VO2kX_fgfUd0x3mBS9NjLUWGZu794WbTepBel3cBg08/edit) that includes germline filtering.
* SangerWES_AC: in Sanger WES
* SangerRecalibWES_AC: in Sanger WES after realignment at Broad
* RNAseq_AC: in Broad RNAseq data from the CCLE2 project
* HC_AC: in Broad Hybrid capture data from the CCLE2 project
* RD_AC: in Broad Raindance data from the CCLE2 project
* WGS_AC: in Broad WGS data from the CCLE2 project

Additional columns:
 
* isTCGAhotspot: is this mutation commonly found in TCGA
* TCGAhsCnt: number of times this mutation is observed in TCGA
* isCOSMIChotspot: is this mutation commonly found in COSMIC
* COSMIChsCnt: number of samples in COSMIC with this mutation
* ExAC_AF: the allelic frequency in the Exome Aggregation Consortium (ExAC)

Descriptions of the remaining columns in the MAF can be found here: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/

#### Mutations
CCLE mutation calls are aggregated from several different sources and sequencing technologies.

Currently, WES-based calls are generated on a quarterly basis using a cell line implementation of the CGA WES Characterization Pipeline (more info here: https://docs.google.com/document/d/1VO2kX_fgfUd0x3mBS9NjLUWGZu794WbTepBel3cBg08/edit). The cell line implementation of this pipeline has one key difference: because CCLE lines do not have a matched control sample, we use the same normal sample as a matched normal input for all samples using the same baits (Agilent vs. ICE). In addition, mutation calls still use hg19 alignment.

Quarterly WES-based mutation calls are added to the existing mutation calls from previous releases. WGS, Sanger WES, RNAseq, HC, and RD based calls from the 2nd phase of the CCLE project are also included. More details on how these mutations were called and filtered can be found in the manuscript “Next generation characterization of the Cancer Cell Line Encyclopedia” in Nature.

#### DepMap Public 20Q2
This DepMap release contains data from CRISPR knockout screens from project Achilles, as well as genomic characterization data from the CCLE project.

#### Data Usage
Please cite the following when using these data
> Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

This data was generated in part from funding from
This project is partially funded by CTD2, the Achilles consortium, and The Carlos Slim Foundation in Mexico through the Slim Initiative for Genomic Medicine.

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

- - - 

### CCLE_gene_cn.csv (Renamed as CCLE_GCN.csv)
* 27639 Genes
* 1745 Cell Lines
* 35 Primary Diseases
* 37 Lineages

#### File Description
Gene level copy number data, log2 transformed with a pseudo count of 1. This is generated by mapping genes onto the segment level calls.

* Rows: cell lines (Broad IDs)
* Columns: genes (HGNC symbol and Entrez ID)

#### Copy number
The CN is called using Broad WES as top priority. If Broad WES is not yet available for a line, then Sanger WES data is used as long as the Sanger WES CN data correlates well with the Achilles log-fold change data. If the correlation is too low and the Sanger WES CN profile differs from the Broad Affy CN, then we used Broad Affy CN. If no WES data is available for a cell line, Broad Affy data is used. We plan to complete WES for all lines that do not have WES soon so we will phase out using Affy CN data. For the WES data, CN is called using a panel of normals that is female only. Therefore X CN is relative to 2 copies of X for WES data. We do not use the X chromosome CN data for cell lines where CN was derived from Affy arrays.

CCLE WES copy number data is generated by running the GATK copy number pipeline aligned to hg38. Tutorials and descriptions of this method can be found here https://software.broadinstitute.org/gatk/documentation/article?id=11682, https://software.broadinstitute.org/gatk/documentation/article?id=11683. WES samples have been realigned to hg38 and run through this pipeline.

Post processing of the segmented level data addressed gaps and the ends of the called chromosomal regions.

#### DepMap Public 20Q2
Please cite the following when using these data. For this DepMap release: 

> DepMap, Broad (2020): DepMap 20Q2 Public. figshare. Dataset. https://doi.org/10.6084/m9.figshare.12280541.v4.
This DepMap release contains data from CRISPR knockout screens from project Achilles, as well as genomic characterization data from the CCLE project.

#### Data Usage
Please cite the following when using these data
> Mahmoud Ghandi, Franklin W. Huang, Judit Jané-Valbuena, Gregory V. Kryukov, ... Todd R. Golub, Levi A. Garraway & William R. Sellers. 2019. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).

This data was generated in part from funding from
This project is partially funded by CTD2, the Achilles consortium, and The Carlos Slim Foundation in Mexico through the Slim Initiative for Genomic Medicine.

#### Terms and conditions
CCLE publishes it's data under the Terms and Conditions linked [here](https://depmap.org/portal/ccle/terms_and_conditions).

- - - 

### Achilles_gene_effect.csv (Renamed as Achilles_GeneEssentiality.csv)
* 18119 Genes
* 769 Cell Lines
* 29 Primary Diseases
* 26 Lineages

#### File Description
Post-CERES

CERES data with principle components strongly related to known batch effects removed, then shifted and scaled per cell line so the median nonessential KO effect is 0 and the median essential KO effect is -1.

* Columns: genes in the format “HUGO (Entrez)”
* Rows: cell lines (Broad IDs)

#### Achilles
This Achilles dataset contains the results of genome-scale CRISPR knockout screens for 18,119 genes in 769 cell lines. The dataset was processed using the following steps:
* Sum raw readcounts by replicate and guide
* Remove the list of guides with suspected off-target activity
* Remove guides with pDNA counts less than one millionth of the pDNA pool
* Remove replicates that fail fingerprinting match to parent or derivative lines
* Remove replicates with total reads less than 15 million
* Calculate reads per million, add pseudo-count of 1, then log2-fold-change from pDNA counts for each replicate
* Calculate the NNMD for each replicate using genes targeting the Hart reference non-essentials and the intersection of the Hart and Blomen essentials, and remove those with values more positive than -1.0. See Hart et al., Mol. Syst. Biol, 2014 and Blomen et al., Science, 2015
* Remove replicates that do not have a Pearson coefficient > .61 with at least one other replicate for the line when looking at genes with the highest variance (top 3%) in gene effect across cell lines. Equivalent to excluding lines whose replicates have a p-value greater than 0.05 of having at least as high a correlation with a random line as with each other.
* Calculate the NNMD for each cell line after averaging remaining replicates, and remove those more positive than -1.0
* Run CERES to generate gene-level scores
* Scale so the median of common essentials in each cell lines is -1
* Remove the mean and variance of each gene across datasets, so each gene has mean 0 and variance 1
* Remove the first five principle components of the resulting matrix and restore the prior means and variances of genes
* Scale again so the median of common essentials in each cell lines is -1
* Identify pan-dependent genes as those for whom 90% of cell lines rank the gene above a given dependency cutoff. The cutoff is determined from the central minimum in a histogram of gene ranks in their 90th percentile least dependent line
* For each CERES gene score, infer the probability that the score represents a true dependency or not. This is done using an EM step until convergence independently in each cell line. The dependent distribution is given by the list of essential genes. The null distribution is determined from unexpressed gene scores in those cell lines that have expression data available, and from the Hart non-essential gene list in the remainder
* Replace all genes on X chromosome for cell lines that only have Broad SNP copy number data with NA in gene_effect.csv, gene_effect_unscaled.csv, and gene_dependency.csv
* 1,309 genes are NA-ed in 12 cell lines due to identified contamination. New data for these cell lines will be generated in the future
The source for copy number data varies by cell line. Copy number data indicated as "Sanger WES" are based on the Sanger Institute whole exome sequencing data (COSMIC: http://cancer.sanger.ac.uk/cell_lines, EGA accession number: EGAD00001001039) reprocessed using CCLE pipelines. Copy number source was chosen according to the following logic:

Broad WES for lines where available
Broad SNP when Broad WES is not available and Sanger WES not available, or Sanger WES copy number has less correlation with logfold change than Broad SNP
Sanger WES in all other cases

#### Data Usage
Please cite the following when using these data
For this DepMap release:
> DepMap, Broad (2020): DepMap 20Q2 Public. figshare. Dataset. https://doi.org/10.6084/m9.figshare.12280541.v4.

For CRISPR datasets:
> Robin M. Meyers, Jordan G. Bryan, James M. McFarland, Barbara A. Weir, ... David E. Root, William C. Hahn, Aviad Tsherniak. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nature Genetics 2017 October 49:1779–1784. doi:10.1038/ng.3984

- - - 

### sample_info.csv
* 1804 Cell Lines
* 35 Primary Diseases
* 38 Lineages

##### File Description
* DepMap_ID: Static primary key assigned by DepMap to each cell line
* stripped_cell_line_name: Cell line name with alphanumeric characters only
* CCLE_Name: Previous naming system that used the stripped cell line name followed by the lineage; no longer assigned to new cell lines
* alias: Additional cell line identifiers (not a comprehensive list)
* COSMIC_ID: Cell line ID used in Cosmic cancer database
* lineage, lineage_subtype, lineage_sub_subtype, lineage_molecular_subtype: Cancer type classifications in a standardized form
* sex: Sex of tissue donor if known
* source: Source of cell line vial used by DepMap
* Achilles_n_replicates: Number of replicates used in Achilles CRISPR screen passing QC
* cell_line_NNMD: Difference in the means of positive and negative controls normalized by the standard deviation of the negative control distribution
* culture_type: Growth pattern of cell line (Adherent, Suspension, Mixed adherent and suspension, 3D, or Adherent (requires laminin coating))
* culture_medium: Medium used to grow cell line
* cas9_activity: Percentage of cells remaining GFP positive on days 12-14 of cas9 activity assay as measured by FACs
* RRID: Cellosaurus research resource identifier
* sample_collection_site: Tissue collection site
* primary_or_metastasis: Indicates whether tissue sample is from primary or metastatic site
* disease: General cancer lineage category
* disease_subtype: Subtype of disease; specific disease name
* age: If known, age of tissue donor at time of sample collection
* Sanger_model_ID: Sanger Institute Cell Model Passport ID
* additional_info: Further information about cell line modifications and drug resistance

This data was generated in part from funding from
This project is partially funded by CTD2, the Achilles consortium, and The Carlos Slim Foundation in Mexico through the Slim Initiative for Genomic Medicine.

#### Terms and conditions
DepMap publishes it's data under CC Attribution 4.0 license

- - - 
